Common/Common.csproj

```diff
@@ -6,18 +6,18 @@
     <RootNamespace>Wokhan.WindowsFirewallNotifier.Common</RootNamespace>
     <ApplicationRevision>0</ApplicationRevision>
     <ApplicationVersion>1.0.0.%2a</ApplicationVersion>
+    <NeutralLanguage>en-US</NeutralLanguage>
     <Description>Windows Firewall Notifier - Common Library</Description>
     <Product>Windows Firewall Notifier</Product>
     <Copyright>2011-2020 Wokhan (https://github.com/wokhansoft/WFN)</Copyright>
     <Authors>Wokhan Solutions</Authors>
     <OutputPath>..\bin\$(ConfigurationName)</OutputPath>
+    <Nullable>Enable</Nullable>

```

Comment:
I assume, this is used for checking null parameters during compile time?

Reply:
More than just compile time. It's also used in VS to warn us about unexpected possible null assignment/usage, as I mentioned before: it's really powerful, in fact this is how nulls should have been handled since the "beginning" (i.e. purposely having nulls when it makes sense, and even having errors when you return null or use a possibly null value and shouldn't).
_In fine_, it should even remove any need to actually check for unexpected null values since it won't be possible... You'll have either possibly-null reference, or never-null reference.